packages/fluentui/react-icons-northstar/src/components/CallTransferredIcon.tsx

```diff
@@ -0,0 +1,110 @@
+import * as React from 'react';
+import cx from 'classnames';
+import { createSvgIcon } from '../utils/createSvgIcon';
+import { iconClassNames } from '../utils/iconClassNames';
+
+export const CallTransferredIcon = createSvgIcon({
```

Comment:
![image](https://user-images.githubusercontent.com/14183168/87543471-1e641380-c6a5-11ea-9714-667784b0de0f.png)

Does it have correct look?

Reply:
Does not go into icons. Remains as an Illustration in TMP